acados/ocp_nlp/ocp_nlp_constraints_bgp.c

```diff
@@ -0,0 +1,1253 @@
+    }
+
+    if (opts->compute_hess)
+    {
+        printf("\nerror: compute_hess!=0 not supported (yet) in ocp_nlp_constraints_bgp\n");
+        exit(1);
+    }
+
+    return;
+}
+
+
+
+void ocp_nlp_constraints_bgp_compute_fun(void *config_, void *dims_, void *model_, void *opts_, void *memory_, void *work_)
+{
+	printf("\nerror: ocp_nlp_constraints_bgp_compute_fun: not implemented yet\n");
+	exit(1);
+}
+
+
+
+void ocp_nlp_constraints_bgp_config_initialize_default(void *config_)
+{
+    ocp_nlp_constraints_config *config = config_;
+
+    config->dims_calculate_size = &ocp_nlp_constraints_bgp_dims_calculate_size;
+    config->dims_assign = &ocp_nlp_constraints_bgp_dims_assign;
+    config->dims_initialize = &ocp_nlp_constraints_bgp_dims_initialize;
+    config->dims_set = &ocp_nlp_constraints_bgp_dims_set;
+    config->dims_get = &ocp_nlp_constraints_bgp_dims_get;
+    config->model_calculate_size = &ocp_nlp_constraints_bgp_model_calculate_size;
+    config->model_assign = &ocp_nlp_constraints_bgp_model_assign;
+    config->model_set = &ocp_nlp_constraints_bgp_model_set;
+    config->opts_calculate_size = &ocp_nlp_constraints_bgp_opts_calculate_size;
+    config->opts_assign = &ocp_nlp_constraints_bgp_opts_assign;
+    config->opts_initialize_default = &ocp_nlp_constraints_bgp_opts_initialize_default;
+    config->opts_update = &ocp_nlp_constraints_bgp_opts_update;
+    config->opts_set = &ocp_nlp_constraints_bgp_opts_set;
+    config->memory_calculate_size = &ocp_nlp_constraints_bgp_memory_calculate_size;
+    config->memory_assign = &ocp_nlp_constraints_bgp_memory_assign;
+    config->memory_get_fun_ptr = &ocp_nlp_constraints_bgp_memory_get_fun_ptr;
+    config->memory_get_adj_ptr = &ocp_nlp_constraints_bgp_memory_get_adj_ptr;
+    config->memory_set_ux_ptr = &ocp_nlp_constraints_bgp_memory_set_ux_ptr;
+    config->memory_set_tmp_ux_ptr = &ocp_nlp_constraints_bgp_memory_set_tmp_ux_ptr;
+    config->memory_set_lam_ptr = &ocp_nlp_constraints_bgp_memory_set_lam_ptr;
+    config->memory_set_tmp_lam_ptr = &ocp_nlp_constraints_bgp_memory_set_tmp_lam_ptr;
+    config->memory_set_DCt_ptr = &ocp_nlp_constraints_bgp_memory_set_DCt_ptr;
+    config->memory_set_RSQrq_ptr = &ocp_nlp_constraints_bgp_memory_set_RSQrq_ptr;
+    config->memory_set_z_alg_ptr = &ocp_nlp_constraints_bgp_memory_set_z_alg_ptr;
+    config->memory_set_dzdux_tran_ptr = &ocp_nlp_constraints_bgp_memory_set_dzduxt_ptr;
+    config->memory_set_idxb_ptr = &ocp_nlp_constraints_bgp_memory_set_idxb_ptr;
+    config->memory_set_idxs_ptr = &ocp_nlp_constraints_bgp_memory_set_idxs_ptr;
+    config->workspace_calculate_size = &ocp_nlp_constraints_bgp_workspace_calculate_size;
+    config->initialize = &ocp_nlp_constraints_bgp_initialize;
+    config->update_qp_matrices = &ocp_nlp_constraints_bgp_update_qp_matrices;
+    config->compute_fun = &ocp_nlp_constraints_bgp_compute_fun;
+    config->config_initialize_default = &ocp_nlp_constraints_bgp_config_initialize_default;
+
+
+    return;
+}
```

Comment:
It is nearly impossible to check the changes to the file, so ok on trust ;)

Reply:
I don't know what happens with with git when we have to fix PRs locally. It would be great to understand what is going on :) Anyway, if you checkout the PR locally and rename `ocp_nlp_constraints_bgp.h -> ocp_nlp_constraints_bghp.c` and do `git diff 00071c4013af0d -- ocp_nlp_constraints_bghp.c` you can see all the changes (same for `ocp_nlp_constraints_bgp.h`)